src/jira/client/jira-client.ts

```diff
@@ -414,13 +414,16 @@ const batchedBulkUpdate = async (
 	return Promise.all(batchedUpdates);
 };
 
+const findIssueKeyAssociation = (resource: IssueKeyObject): JiraAssociation | undefined =>
+	resource.associations?.find(a => a.associationType == "issueIdOrKeys");
```

Comment:
Will it be possible to multiple entries in the `associations` that have type `issueIdOrKeys`?
Not sure what's the spec for this api say.  Couldn't find anything on doc.

Reply:
There should only be 1 association per type, but also, we're the ones creating it in the first place.  Having to support multiple is much more difficult and unnecessary at this moment.